robottelo/hosts.py

```diff
@@ -2166,6 +2173,37 @@ def enroll_ad_and_configure_external_auth(self, ad_data):
             self.execute('systemctl daemon-reload && systemctl restart httpd.service').status == 0
         )
 
+    def generate_inventory_report(self, org):
+        """Function to perform inventory upload."""
+        generate_report_task = 'ForemanInventoryUpload::Async::UploadReportJob'
+        timestamp = datetime.utcnow().strftime('%Y-%m-%d %H:%M')
+        self.api.Organization(id=org.id).rh_cloud_generate_report()
+        wait_for(
+            lambda: self.api.ForemanTask()
+            .search(query={'search': f'{generate_report_task} and started_at >= "{timestamp}"'})[0]
+            .result
+            == 'success',
+            timeout=400,
+            delay=15,
+            silent_failure=True,
+            handle_exception=True,
+        )
+
+    def sync_inventory_status(self, org):
+        """Perform inventory sync"""
+        inventory_sync = self.api.Organization(id=org.id).rh_cloud_inventory_sync()
+        wait_for(
+            lambda: self.api.ForemanTask()
+            .search(query={'search': f'id = {inventory_sync["task"]["id"]}'})[0]
+            .result
```

Comment:
join with the line above (line limit is 99 chars)

Reply:
pre-commit is failing if I do this.